docs/source/_json/addons_install.resp

```diff
@@ -0,0 +1,6 @@
+HTTP/1.1 200 OK
```

Comment:
We might want to mirror the behavior of [`POST` for content CRUD operations](http://plonerestapi.readthedocs.io/en/latest/content.html#successful-response-204-no-content) here:
- Return `204 No Content` to just acknowledge success (we don't really have anything else to say here)
- Return `200 OK` with the new representation of the "addon resource" if the client requests it via the `Prefer: return=representation` header

I think the same should be applicable to `uninstall` and `upgrade`.